app/assets/stylesheets/ui_components/select.scss

```diff
@@ -0,0 +1,8 @@
+.form-inline [data-react-class="ui_components.Select"] {
+  display: inline-block;
+  vertical-align: middle;
+
+  .search-field > input {
+    min-width: 222px;
```

Comment:
Bootstrap just seems to specify it as `auto`. When I do that with chosen, it's about 1em wide. If I don't specify it at all, it tries to figure out how wide the containing element is and fails gloriously at that. The filed is then something over 1500px wide...